go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ElrondNetwork/elastic-indexer-go
 go 1.17
 
 require (
-	github.com/ElrondNetwork/elrond-go-core v1.1.16-0.20220414130405-e3cc29bc7711
+	github.com/ElrondNetwork/elrond-go-core v1.1.20-0.20220921103015-577300303692
```

Comment:
proper release

Maybe create branch `rc/v1.4.0` and retarget this PR to that branch?